Data/Sector/Hong-Kong-Sector-File.ese

```diff
@@ -2779,6 +2432,96 @@ ACTIVE:VMMC:16
 DEPAPT:VHHH
 ARRAPT:VMMC
 
+;APP/DEP 07s NAP
+SECTOR:VHHK·VHHK_SCT_07NAP_APP_01·000·135:00000:13500
+OWNER:APP:FAD:DEP:DEH:TRE:PAR
+ALTOWNER:Mentoring VHHH_APP:INS:MTR
+ALTOWNER:Mentoring HKG_E_CTR:APP:FAD:DEP:DEH:INS:MTR
+BORDER:112a:113:110
+ACTIVE:VHHH:NAP
+DEPAPT:VHHH
+ARRAPT:VHHH
+
+SECTOR:VHHK·VHHK_SCT_07NAP_APP_02·000·165:00000:16500
+OWNER:APP:FAD:DEP:DEH:TRE:PAR
+ALTOWNER:Mentoring VHHH_APP:INS:MTR
+ALTOWNER:Mentoring HKG_E_CTR:APP:FAD:DEP:DEH:INS:MTR
+BORDER:112:111:21:22:23:24
+ACTIVE:VHHH:NAP
+DEPAPT:VHHH
+ARRAPT:VHHH
+
+SECTOR:VHHK·VHHK_SCT_07NAP_DEP_01·000·135:00000:13500
+OWNER:DEP:DEH:APP:FAD:PAR:TRE
+ALTOWNER:Mentoring VHHH_APP:DEP:DEH:INS:MTR
+ALTOWNER:Mentoring HKG_E_CTR:DEP:DEH:APP:FAD:INS:MTR
+BORDER:115:110:114:26:16:117
+ACTIVE:VHHH:NAP
+DEPAPT:VHHH
+ARRAPT:VHHH
+
+SECTOR:VHHK·VHHK_SCT_07NAP_DEP_02·000·165:00000:16500
+OWNER:DEP:DEH:APP:FAD:PAR:TRE
+ALTOWNER:Mentoring VHHH_APP:DEP:DEH:INS:MTR
+ALTOWNER:Mentoring HKG_E_CTR:DEP:DEH:APP:FAD:INS:MTR
+BORDER:116:117:109a:28
+ACTIVE:VHHH:NAP
+DEPAPT:VHHH
+ARRAPT:VHHH
+
+SECTOR:VHHK·VHHK_SCT_07NAP_DEP_03·000·250:00000:25000
+OWNER:DEP:DEH:APP:FAD:PAR:TRE
+ALTOWNER:Mentoring VHHH_APP:DEP:DEH:INS:MTR
+ALTOWNER:Mentoring HKG_E_CTR:DEP:DEH:APP:FAD:INS:MTR
+BORDER:111:115:116:32:35:36:37
+ACTIVE:VHHH:NAP
+DEPAPT:VHHH
+ARRAPT:VHHH
+
+SECTOR:VHHK·VHHK_SCT_07NAP_DEH_01·100·250:10000:25000
+OWNER:DEH:DEP:APP:FAD:TRE:PAR
+ALTOWNER:Mentoring VHHH_APP:DEH:DEP:INS:MTR
+ALTOWNER:Mentoring HKG_E_CTR:DEH:DEP:APP:FAD:INS:MTR
+BORDER:3:4:5:6:7
+ACTIVE:VHHH:NAP
+DEPAPT:VHHH
+ARRAPT:VHHH
+
+SECTOR:VHHK·VHHK_SCT_07NAP_DEH_02·135·250:13500:25000
+OWNER:DEH:DEP:APP:FAD:TRE:PAR
+ALTOWNER:Mentoring VHHH_APP:DEH:DEP:INS:MTR
+ALTOWNER:Mentoring HKG_E_CTR:DEH:DEP:APP:FAD:INS:MTR
+BORDER:112a:113:110
+ACTIVE:VHHH:NAP
+DEPAPT:VHHH
+ARRAPT:VHHH
+
+SECTOR:VHHK·VHHK_SCT_07NAP_DEH_03·135·250:13500:25000
+OWNER:DEH:DEP:APP:FAD:TRE:PAR
+ALTOWNER:Mentoring VHHH_APP:DEH:DEP:INS:MTR
+ALTOWNER:Mentoring HKG_E_CTR:DEH:DEP:APP:FAD:INS:MTR
+BORDER:115:110:114:26:16:117
+ACTIVE:VHHH:NAP
+DEPAPT:VHHH
+ARRAPT:VHHH
+
+SECTOR:VHHK·VHHK_SCT_07NAP_DEH_04·165·250:13500:25000
+OWNER:DEH:DEP:APP:FAD:TRE:PAR
+ALTOWNER:Mentoring VHHH_APP:DEH:DEP:INS:MTR
+ALTOWNER:Mentoring HKG_E_CTR:DEH:DEP:APP:FAD:INS:MTR
+BORDER:116:117:109a:28
+ACTIVE:VHHH:NAP
+DEPAPT:VHHH
+ARRAPT:VHHH
+
+SECTOR:VHHK·VHHK_SCT_07NAP_DEH_05·165·250:13500:25000
+OWNER:DEH:DEP:APP:FAD:TRE:PAR
+ALTOWNER:Mentoring VHHH_APP:DEH:DEP:INS:MTR
+ALTOWNER:Mentoring HKG_E_CTR:DEH:DEP:APP:FAD:INS:MTR
+BORDER:112:111:21:22:23:24
+ACTIVE:VHHH:NAP
+DEPAPT:VHHH
+ARRAPT:VHHH
 
```

Comment:
### (MAJOR PRIORITY)
We need to write new COPX rules for ALL the new noise abatement sectors

Reply:
New COPX:
 07 NAP DEP -> 07 NAP APP 
 9000 GUAVA